### PR TITLE
updating to use new EP params

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**
Dual Merge: [BE PR](https://github.com/MetaPhase-Consulting/State-TalentMAP-API/pull/468)
[Mock PR](https://github.com/MetaPhase-Consulting/mock-fsbid/pull/149)
The mock had already been set up for these. Made sure to test location_city, location_country, and location_state(when also domestic). NVM, re-read ticket; have to update mock.

### CODE REVIEW

File: src/Constants/Sort.js [hunk 1]
```diff
@@ -13,8 +13,8 @@ const POSITION_SEARCH_SORTS$ = {
     { value: '-ted', text: 'TED: Latest' },
     { value: 'position__position_number', text: 'Position number: Low to high' }, // numbers first, then A-Z
     { value: '-position__post__has_service_needs_differential', text: 'Featured positions', availableOnly: true }, // sort by service needs first
-    { value: 'location', text: 'Location: A-Z', nonTandemOnly: true },
-    { value: '-location', text: 'Location: Z-A', nonTandemOnly: true },
+    { value: 'location_city', text: 'Location: A-Z', nonTandemOnly: true },
+    { value: '-location_city', text: 'Location: Z-A', nonTandemOnly: true },
```
Comment (elizabeth-jimenez) [inline src/Constants/Sort.js:17]: these were always defaulting to city sort, so i figured we could keep the Text the same

**Comment by mjoyce91** [comment]
So technically I don't think we need to change anything on the front-end. We just need to update the mapping on our API

**Comment by elizabeth-jimenez** [comment]
> So technically I don't think we need to change anything on the front-end. We just need to update the mapping on our API

yea, just figured if we ever want to sort by country or state, in the future, this sets it up for that? we don't have to tho, ur right

**Comment by mjoyce91** [comment]
@elizabeth-jimenez nah you're right, this is good 👍

**Comment by mjoyce91** [comment]
Looks like you just need to update 📸

**Comment by mjoyce91** [comment]
Do we really need all of these? I'd recommend just sticking with what we originally had.

Screen Shot 2021-02-02 at 3 14 53 PM

**Comment by elizabeth-jimenez** [comment]
> Do we really need all of these? I'd recommend just sticking with what we originally had.
> 
> Screen Shot 2021-02-02 at 3 14 53 PM

Consumers like having options....

**Review by mjoyce91 — APPROVED** [review]

**Pull request merged**